sentences/nl/light_HassTurnOn.yaml

```diff
@@ -9,6 +9,8 @@ intents:
           - "Schakel [alle] <lamp> [in] <area> in"
           - "<doe> (alle | in) <area> <lamp> aan"
           - "Schakel (alle | in) <area> <lamp> in"
+          - "<lamp> aan"
```

Comment:
this sentence is missing an <area> definition
It is not allowed to add sentences to turn on all lights, regardless of an area